### ISSUE TITLE
Stonelands....?

### ISSUE THREAD

**Issue by Steve982950** [ThisTestUser/FarLandsModV2#25]
i tele to farlands and they look like stonelands.......

![Minecraft Forge_ 1 20 11_06_2023 18_02_39](https://github.com/ThisTestUser/FarLandsModV2/assets/128298281/65bc9cb9-7566-4672-803f-075d3a64bc1a)

**Comment by ThisTestUser** [comment]
Have you tried launching the mod with Forge 1.20? It should still work.

**Comment by Steve982950** [comment]
nope, not quite.

**Comment by Steve982950** [comment]
edit: it does... but why did u put the stonelands on here?

![Minecraft Forge_ 1 20 11_06_2023 18_02_39](https://github.com/ThisTestUser/FarLandsModV2/assets/128298281/baae74d4-0fb2-4eca-b897-08ecf463e3a3)

**Comment by ThisTestUser** [comment]
That's an issue that appears starting from 1.19, because of a change in the terrain generation in the overworld. The nether and end  should not be affected.

**Comment by Steve982950** [comment]
oh ok! is it possible to fix it?

**Comment by ThisTestUser** [comment]
No. The only possible use for this mod is to extend the world border, which should still work in this version.

**Comment by Steve982950** [comment]
which mod is it?

**Comment by ThisTestUser** [comment]
This one. Look at the config at .minecraft/configs/farlandsmod.cfg. You can remove the world border there.

**Comment by Steve982950** [comment]
ohhhhhhhh. it will fix the weird stonelands right?

**Comment by ThisTestUser** [comment]
No, because there is no fix. You can disable the far lands in the config though which will make the terrain generate like in vanilla.

**Comment by Steve982950** [comment]
k
